edudoc/delegate-crash-course/delegate_crash_course.md

```diff
@@ -253,23 +253,29 @@ Here is an example of a proper change request corresponding to the list above:
 
 ## Requesting changes to person data
 
-_(updated on January 26, 2019)_
+_(updated on October 15, 2023)_
 
-There are several cases for change requests concerning person data. The Results Team has different requisites depending on different circumstances.
+There are several cases for change requests concerning person data (which involves name, gender, date of birth and citizenship of a competitor). The Results Team has different requirements depending on different circumstances.
 
-There are 2 types of data changes: a fix and an update.
+There are two different types of data changes:
 
-- **Fix**: A correction of incorrect data. These are applied for all the records for the Person, from the very first to the more recent results.
-- **Update**: A change in data where the previous version is still valid in the past. These do not affect old records, and only change per the future. This is what creates a dual profile as seen [here](https://www.worldcubeassociation.org/persons/1982FRID01).
+1. **Fix**: A fix changes a person's data for all past and future competitions and is not visible on a competitor’s profile. This form of data change is commonly applied for:
+- Correction of previously incorrect data.
+- Minor nickname changes (e.g. removal/addition of middle names, changing a legal first name to a reasonable nickname, …)
+- Changes of sensitive nature where displaying the previous data is understandably not desirable (e.g. name/gender changes related to gender identity)
+
+2. **Update**: An update internally creates a new sub-profile, so that the previous data remains unchanged for all past competitions and is also shown on the competitor’s profile. One famous example of this can be found [here](https://www.worldcubeassociation.org/persons/1982FRID01). This form of data change is commonly applied for:
```

Comment:
I think we should remove the link to Jessica's profile. Seems like we're drawing attention to a specific individual unnecessarily, people can find examples themselves or ask someone who knows of an example.

Reply:
While this document is accessible publicly, the purpose is to make the Delegates able to understand the difference in these displayed changes. An example is by far the best way to do this, instead of letting new Delegates try and figure it out for themselves, which undermines the whole principle.